akka-stream-tests/src/test/java/akka/stream/javadsl/SourceTest.java

```diff
@@ -328,6 +331,20 @@ public void mustBeAbleToUseToFuture() throws Exception {
     assertEquals("A", result);
   }
 
+  @Test
+  public void mustBeAbleToUseSingle() throws Exception {
+    //#source-single
+    CompletionStage<List<String>> future = Source.single("A").runWith(Sink.seq(), materializer);
+    CompletableFuture<List<String>> completableFuture = future.toCompletableFuture();
+    List<String> result = completableFuture.get();
```

Comment:
Given this is in the docs we should put a comment saying calling get() isn't a good idea for production code and just an example

Reply:
Or put the toCompletableFuture and  get() not in the docs (outside the //#)

Reply:
Given that it is how you can access the resulting list, then maybe do something like `.thenAccept(list -> System.out.println("collected elements: " + list)`

Reply:
Oh the dichotomy of sample code in doc and that which must be unit tested!
Made the changes as suggested.